tests/functional_tests/t0_main/imports/12-batch12.py

```diff
@@ -25,8 +25,16 @@
   - :op:contains:
     - :wandb:runs[0][telemetry][1]  # imports init
     - 82  # mmengine
+  - :op:contains:
+    - :wandb:runs[0][telemetry][1]  # imports init
+    - 83  # diffusers
+  - :op:contains:
+    - :wandb:runs[0][telemetry][1]  # imports init
+    - 84  # evaluate
```

Comment:
Note that the import tests were moved to nightly, so to test, you'd need to manually trigger nightly.